### DISCUSSION TITLE
Not getting events in HA

### DISCUSSION THREAD

**Discussion by papadi** [SukramJ/aiohomematic#1066] [Q&A] [answered]
Hi. I read as much as I could for this but still no luck. Here's my configuration:

1. Running HAOS on a PC
2. Running RaspberryMatic on a RPi4 (with USB dongle for wireless communication and a HmIPW-DRAP for a wired devices)
3. Installed the Homematic(IP) Local integration and received the list of HmIP devices detected.

The problem is that I receive no events either from a wireless HmIP-WRC6 or from a wired HmIPW-WRC6. I tried to set up some automation but also used the developer tools to listen to homematic.keypress events but I get complete silence.

I'm not sure if and how I should configure the callback hostname or callback XML-RPC port or json-rpc port. Not sure if they are relevant. I tried the default (no values at all) and also put the HAOS PC IP as callback host, but nothing worked.

Any ideas?

**Comment by SukramJ** [comment, marked as answer]
Read the [docs](https://github.com/danielperna84/custom_homematic#events-for-homematicip-devices).
